package.json

```diff
@@ -18,8 +18,8 @@
     "@capacitor/keyboard": "5.0.6",
     "@capacitor/status-bar": "5.0.6",
     "@ionic/storage": "^4.0.0",
-    "@ionic/vue": "^7.0.0",
-    "@ionic/vue-router": "^7.0.0",
+    "@ionic/vue": "^7.6.2-dev.11704729332.13b412c8",
+    "@ionic/vue-router": "^7.6.2-dev.11704729332.13b412c8",
```

Comment:
Is there a Jira ticket that tracks the removal of the dev builds?

Reply:
Yup: https://ionic-cloud.atlassian.net/browse/FW-5873